plenum/test/view_change/test_different_last_ordered_before_view_change.py

```diff
@@ -0,0 +1,154 @@
+import pytest
+import sys
+
+from plenum.server.node import Node
+from plenum.test.delayers import cDelay, pDelay, ppDelay
+from plenum.test.helper import sdk_send_random_and_check, \
+    sdk_send_random_requests, sdk_get_replies
+from plenum.test.stasher import delay_rules
+from stp_core.loop.eventually import eventually
+
+
```

Comment:
Please add tests that make sure we don't request any Propagates after View Change if one of the replicas on backup instance ordered less than others, and it becomes a Primary

Reply:
Added check that no one propagate request wasn't sent.